### PR TITLE
Add configure docs to docsv2 project

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Moving into configuration. Two documents for Cloud-specific configurations, and then another two documents for node configuration basis and improving security of nodes.

> See netdata/netdata-learn-docusaurus#285 for additional context.
>
> This page contains some elements that will only appear on the deployed Learn site, so see the deploy preview for this page for the full picture: https://deploy-preview-285--netdata-docusaurus.netlify.app/docs/get
>
> Some links will not work, as the target docs do not exist yet.
>
> For now, these PRs will get merged into the docsv2 but not go live on Netdata Learn. Once they're all available, or at least a workable majority, we'll merge them into master and deploy the project as a whole.

##### Component Name

area/docs

### CODE REVIEW

File: docs/configure/spaces-war-rooms.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+> account. If you're not interested in Netdata Cloud's features, you can skip ahead to [node configuration
+> basics](/docs/configure/nodes.md).
+
+## Spaces
+
+Spaces are high-level containers to help you organize your team members and the nodes they can view in each War Room.
+You already have at least one Space in your Netdata Cloud account.
+
+To create a new Space, click the **+** icon, enter its name, and click **Save**. Netdata Cloud distinguishes between
+Spaces with an appreviated version of each's name, and you can click on any of the icons to switch between them.
```
Comment (aabatangle) [inline docs/configure/spaces-war-rooms.md:26]: appreviated --> abbreviated? also, "each's" is a little confusing to me.

File: docs/configure/spaces-war-rooms.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+The organization you choose will likely be based on two factors:
+
+1.  The fact that any node can be claimed to a single Space.
+2.  The size of your team and the complexity of the infrastructure you monitor.
+
+A single Space puts all your metrics in one easily-accessible place, while multiple Spaces creates logical division
+between different users and different pieces of a large infrastructure.
+
+For example, a large organization might have one SRE team for the user-face SaaS application, and a second IT team for
```
Comment (aabatangle) [inline docs/configure/spaces-war-rooms.md:39]: user-facing?

File: docs/configure/spaces-war-rooms.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+managing employee's hardware. Since these teams don't monitor the same nodes, they can work in separate Spaces and then
```
Comment (aabatangle) [inline docs/configure/spaces-war-rooms.md:40]: employees'

**Review by aabatangle — CHANGES_REQUESTED** [review]

File: docs/configure/invite-collaborate.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Invite new users by clicking on your Space's name in the top navigation, and then **Invite more users**, to open the
+invitation pane.
+
+![Opening and navigating the invitation
+panel](https://user-images.githubusercontent.com/1153921/92025596-a618e680-ed14-11ea-9c1f-a61fdcb8aa4e.png)
+
+Enter their email address and name. They can change this name once they accept your invitation.
+
+Choose which War Rooms you want to add this user to, then click the plus **+** button to add the invitation to the
```
Comment (jlbriston) [inline docs/configure/invite-collaborate.md:24]: Admins manage user permissions and have control over who can access specific Spaces and War Rooms.

File: docs/configure/invite-collaborate.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+## Invite new members
+
+Invite new users by clicking on your Space's name in the top navigation, and then **Invite more users**, to open the
+invitation pane.
+
+![Opening and navigating the invitation
+panel](https://user-images.githubusercontent.com/1153921/92025596-a618e680-ed14-11ea-9c1f-a61fdcb8aa4e.png)
+
+Enter their email address and name. They can change this name once they accept your invitation.
+
```
Comment (jlbriston) [inline docs/configure/invite-collaborate.md:23]: Admins manage user permissions and have control over who can access specific Spaces and War Rooms.

Comment (jlbriston) [inline docs/configure/invite-collaborate.md:23]:
```suggestion
Admins manage user permissions and have control over who can access specific Spaces and War Rooms.

```

File: docs/configure/invite-collaborate.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+Choose which War Rooms you want to add this user to, then click the plus **+** button to add the invitation to the
+**New invitations to be sent** queue. Repeat the process with everyone you want to invite to your Space.
+
+When you're ready to send the new invitations you created, hit the **Send** button. Netdata Cloud sends these
+invitations and moves them to the **Invitations awaiting response** category.
+
+Your team will receive their invitations momentarily and prompt them to sign in to join your Space.
```
Comment (jlbriston) [inline docs/configure/invite-collaborate.md:30]:
```suggestion
Your team will receive their email invitations momentarily and prompt them to sign in to join your Space.
```

File: docs/configure/invite-collaborate.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+the metrics, alarm statuses, dashboards, and people in one place is a powerful asset for any infrastructure monitoring
+team.
+
+Assets like dashboards and bookmarks are shared between members of a War Room. As soon as one member creates a
+dashboard, for example, other members of the same War Room can see it in the War Room's dropdown and supplement it with
+additional charts/text.
+
+Let's say you get an alert from your nodes about an excess of 500-type errors in your Nginx logs. Your team can hop on a
+Slack call to begin working together. While one engineer handles creating a new dashboard with a half-dozen relevant
+Nginx log metrics, another can dive into the real-time node dashboard investigate correlated charts in granular detail.
```
Comment (jlbriston) [inline docs/configure/invite-collaborate.md:44]:
```suggestion
Nginx log metrics, another can dive into the real-time node dashboard and investigate correlated charts in granular detail.
```

File: docs/configure/invite-collaborate.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+## What's next?
+
+If your team members have trouble signing in, direct them to the [Netdata Cloud sign in
+doc](https://learn.netdata.cloud/docs/cloud/manage/sign-in). Or, find answers to other common questions about Netdata
+Cloud in our [FAQ](https://learn.netdata.cloud/docs/cloud/faq-glossary).
+
+Next, we recommend you learn the [basics of node configuration](/docs/configure/nodes.md). While the Netdata Agent is
+proudly zero-configuration in most cases, you should understand how to tweak its settings to give you the best Netdata,
+for example, to increase metrics retention and improve security.
```
Comment (jlbriston) [inline docs/configure/invite-collaborate.md:54]:
```suggestion
for example, to increase metrics retention with [Netdata's Database](hhttps://learn.netdata.cloud/docs/agent/database/engine) engine or improve security.
```

Comment (jlbriston) [inline docs/configure/invite-collaborate.md:54]: Perhaps the "Improve security" can link to the "secure your nodes" documentation

**Review by jlbriston — APPROVED** [review]

**Pull request merged**